dts/bindings/riscv/nordic,vpr-coprocessor.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+compatible: "nordic,vpr-coprocessor"
+
+description: |
+  VPR coprocessor
+
+  VPR is a RISC-V CPU implementation. VPR instances are exposed to other CPUs as
+  peripherals. A domain can take ownership of one or more VPRs.
+
+include: base.yaml
+
+properties:
+  cpu:
+    type: int
+    required: true
+    description: |
+      Processor ID of the VPR core.
```

Comment:
Is it used anywhere?

Reply:
IIRC by regtool @57300 ?

Reply:
Then it should be optional, it's not needed in nRF54L.

Reply:
we should probably add a link to the CPU node instead, this duplicates unnecessary information, but I assume this requires fixes to regtool...

Reply:
Can't link the CPU node, because it has to be deleted.

Reply:
Let's make it optional. I wasn't aware that nRF54L doesn't define Processor IDs.

Reply:
Done.